README.md

```diff
@@ -23,14 +22,26 @@
 -   `DEFAULT_SMTP_TLS_REJECT_UNAUTHORIZED`: SMTP config `tls.rejectUnauthorized` for nodemailer, config would open a connection to TLS server with self-signed or invalid TLS certificate
 -   `DEFAULT_SMTP_AUTH_METHOD`: SMTP config `authMethod` for nodemailer, defines preferred authentication method, defaults to ‘PLAIN’
 -   `DEFAULT_SMTP_DEBUG`: SMTP config `debug` for nodemailer, if set to true, then logs SMTP traffic, otherwise logs only transaction events
+-   `UI_URL`: host for ui
 
 ## FILE STORAGE API
 
+-   `STORAGE`: Can be `s3` or `fs` for files and images uploads
+-   `BLOB_URL`: (optional) if use external server for file upload
```

Comment:
`BLOB_URL`: (optional) if use external server for file upload. if `STORAGE` is `s3` you must set a BLOB_URL